subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -29,3 +29,9 @@ zephyr_library_link_libraries_ifdef(
 zephyr_library_link_libraries(subsys__bluetooth)
 
 zephyr_include_directories(.)
+
+# This definition needs to be set globally as it is supposed to be used
+# in nrfx_glue.h also when other libraries are built.
+zephyr_compile_definitions(
```

Comment:
Depending on https://github.com/zephyrproject-rtos/zephyr/pull/43748#issuecomment-1066600433, we may need two header files

* `mpsl_used_resources.h` and `sdc_used_resources.h`